cli/src/main.rs

```diff
@@ -474,24 +481,24 @@ fn run() -> Result<(), CliError> {
                         Arg::with_name("url")
                             .short("U")
                             .long("url")
-                            .help("The URL of the Splinter daemon REST API")
+                            .help("URL of the Splinter daemon REST API")
                             .takes_value(true),
                     )
                     .arg(
                         Arg::with_name("management_type")
                             .long("management-type")
                             .long_help(
```

Comment:
I think most or all of these can be changed to just `help` instead of `long_help` since they're all short now.

Reply:
Addressed